.gitignore

```diff
@@ -78,3 +78,4 @@ target/
 
 # IDEs
 .vscode
+.idea/
```

Comment:
It's from my IDE

Reply:
Ah, I see. Looks like we have it in a few other repos as well 👍 . 

Just in case you're not aware: you can also setup a global `~/.gitignore`, so each developer can ignore all the junk files that are specific to the software they're using. I've got one setup for a few of types of file.